frontend/src/components/fields/MoleculeSelector/index.tsx

```diff
@@ -60,7 +65,10 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
       onInputChange={(_, newInput) =>
         setInput(addSubscriptsToMolecule(newInput.toUpperCase()))
       }
-      renderOption={(molecule) => addSubscriptsToMolecule(molecule)}
+      renderOption={(props, molecule) => {
+        console.log(molecule);
+        return <li {...props}>{addSubscriptsToMolecule(molecule)}</li>;
+      }}
```

Comment:
We don't want the `console.log` on prod here.

```suggestion
      renderOption={(props, molecule) => <li {...props}>{addSubscriptsToMolecule(molecule)}</li>}
```